src/process/chunks.rs

```diff
@@ -0,0 +1,40 @@
+use diesel::r2d2::{ConnectionManager, Pool};
+use diesel::PgConnection;
+use tokio_diesel::AsyncRunQueryDsl;
+use tracing::error;
+
+use crate::models;
+use crate::schema;
+
+/// Saves chunks to database
+pub(crate) async fn process_chunks(
+    pool: &Pool<ConnectionManager<PgConnection>>,
+    chunks: &[near_indexer::IndexerChunkView],
+    block_height: u64,
+) {
+    let chunk_models: Vec<models::chunks::Chunk> = chunks
+        .iter()
+        .map(|chunk| models::chunks::Chunk::from_chunk_view(block_height, chunk))
```

Comment:
UX nit: `from_chunk_view(block_height, ...)` feels counterintuitive to me; I would expect `chunk` to be the first argument, and the rest to be the additional required information from most generic to most specific ones.